src/ol/interaction/extent.js

```diff
@@ -31,6 +31,8 @@ goog.require('ol.style.Style');
  */
 ol.interaction.Extent = function(opt_options) {
 
+  var options = opt_options ? opt_options : {};
```

Comment:
You can alternatively do:

`var options = opt_options || {};`

Looks cleaner in my opinion.